app/src/routers/ProjectsRouter.tsx

```diff
@@ -57,28 +58,10 @@ const ProjectsRouter: React.FC = () => {
         </ProjectsLayout>
       </AppRoute>
 
-      <AppRoute exact path="/admin/projects/:id/surveys/:survey_id/details" layout={ProjectsLayout}>
-        <ProjectsLayout>
-          <SurveyPage />
-        </ProjectsLayout>
-      </AppRoute>
-
-      <AppRoute exact path="/admin/projects/:id/surveys/:survey_id/attachments" layout={ProjectsLayout}>
-        <ProjectsLayout>
-          <SurveyPage />
-        </ProjectsLayout>
-      </AppRoute>
-
-      <AppRoute exact path="/admin/projects/:id/surveys/:survey_id/observations" layout={ProjectsLayout}>
-        <ProjectsLayout>
-          <SurveyPage />
-        </ProjectsLayout>
-      </AppRoute>
-
-      <AppRoute exact path="/admin/projects/:id/surveys/:survey_id/results" layout={ProjectsLayout}>
-        <ProjectsLayout>
-          <SurveyPage />
-        </ProjectsLayout>
+      <AppRoute path="/admin/projects/:id/surveys/:survey_id" layout={ProjectsLayout}>
```

Comment:
Could you add the title prop here? (see `AppRouter.tsx`).

The getTitle function could be exported, or it could just be removed tbh, as there really is no reason to have it broken out. We may as well just hardcode it in the titles. Whatever you prefer.

Reply:
Yeah SIMS - Survey works, whatever the existing pattern is